love/src/components/Layout/Layout.jsx

```diff
@@ -294,18 +297,19 @@ class Layout extends Component {
   /** Returns the corresponding svg based on Observatory Control Location * */
   getObsLocationIcon = (style) => {
     const location = this.getObsLocation();
+    const { controlLocation } = this.props.controlLocation;
 
-    switch (location) {
-      case 'Unknown':
+    switch (controlLocation ? controlLocation.name : 'unknown') {
+      case 'unknown':
         return <UnknownLocationIcon className={style} />;
-      case 'Tucson':
-        return <CactusIcon className={style} />;
-      case 'Base':
+      case 'tucson':
+        return <CactusIcon title="PRUEBA DE TITULO" className={style} />;
```

Comment:
Is correct this title?

Reply:
thanks for noticing this 😅️. This is incorrect, I will remove it.